src/containers/BurgerBuilder/BurgerBuilder.js

```diff
@@ -101,10 +95,10 @@ class BurgerBuilder extends Component {
     }
 
     return (
-      <Aux>
+      <>
         <Modal
-          show={this.state.purchasing}
-          clicked={this.purchaseCancelHandler}
+          isOpen={this.state.purchasing}
+          onDismiss={this.purchaseCancelHandler}
```

Comment:
"on clicked of <Modal/> invoke purchase handler"  is less semantically correct compared to "on dismiss of Modal"